operations/compute_rolling_upgrades/Upgrade_Compute_Nodes_with_CRUS.md

```diff
@@ -32,50 +34,49 @@ Upgrade a set of compute nodes with the Compute Rolling Upgrade Service \(CRUS\)
 
         Example output:
 
-        ```
+        ```toml
         [[results]]
         URI = "/hsm/v2/groups/slurm-nodes/members/x0c0s28b0n0"
         ```
 
-2.  Create a group for upgrading nodes \(upgrading label\).
+1. Create a group for upgrading nodes \(upgrading label\).
 
-    The label name used in this example is upgrading-nodes.
+    The label name used in this example is `upgrading-nodes`.
 
     ```bash
-    ncn-w001# cray hsm groups create --label upgrading-nodes \
-    --description 'Upgrading Node Group for my Compute Node upgrade'
+    ncn# cray hsm groups create --label upgrading-nodes --description 'Upgrading Node Group for my Compute Node upgrade'
```

Comment:
ncn# doesn't seem right. I don't think this will be something you run from a storage node

Reply:
Addressing this with https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4815